### ISSUE TITLE
Can't run make oss on macOS

### ISSUE THREAD

**Issue by Manbeardo** [facebook/sapling#739]
852d3ae added lines to the Makefile that run commands like:

```bash
cp build/scripts-3*/$(HGNAME) $(HGNAME)
```

Globbing outside of a path's basename doesn't appear to be supported in `zsh` or `bash`. Running `make oss` on a macbook with the default shell fails with:

```
cp: build/scripts-3*/hg: No such file or directory
```

**Comment by Manbeardo** [comment]
Globbing might be a red herring. There's no `hg` file in `builds/scripts-3.11`

**Comment by vegerot** [comment]
This might be fixed by #741 ?

**Comment by zzl0** [comment]
closing this issue, it was fixed by https://github.com/facebook/sapling/pull/741 and https://github.com/facebook/sapling/commit/dde8296cfd8dcaa1ad41648d44b7480310953901
